### PR TITLE
Scanning improvements

### PR DESCRIPTION
**Pull request by cbridgha**

### CODE REVIEW

**Comment by cbridgha** [comment]
#build

File: dev/build.depScanner/src/io/openliberty/depScanner/CheckRepoAndProductPOMs.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+/**
+ *
+ */
+public class CheckRepoAndProductPOMs {
+
+    private static List<Dependency> masterPomDepList;
```
Comment (tevans78) [inline dev/build.depScanner/src/io/openliberty/depScanner/CheckRepoAndProductPOMs.java]:
```suggestion
    private static List<Dependency> primaryPomDepList;
```

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#libby

**Review by jimblye — APPROVED** [review]

**Comment by cbridgha** [comment]
#libby

**Pull request merged**